pgcli/main.py

```diff
@@ -81,6 +81,10 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         self.syntax_style = c['main']['syntax_style']
         self.cli_style = c['colors']
         self.wider_completion_menu = c['main'].as_bool('wider_completion_menu')
+
+        on_error_modes = {'STOP': ON_ERROR_STOP, 'RESUME': ON_ERROR_RESUME}
```

Comment:
We don't advertise `RAISE` as a valid option in the config file. So if a user decides to change it to a string that is neither `STOP` nor `RESUME`, pgcli will crash at startup, which I think is a valid behavior. 

The reason it will crash at startup is because the line below this tries to use the string to look up a value from the `on_error_modes` dictionary, which won't have a valid entry for that key. So it will throw a `KeyError` and crash pgcli.